plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetastoreAccessOperations.java

```diff
@@ -219,7 +219,7 @@ public void testSelectFromFreshMaterializedView()
 
         assertMetastoreInvocations("SELECT * FROM test_select_fresh_mview_view",
                 ImmutableMultiset.<MetastoreMethod>builder()
-                        .addCopies(GET_TABLE, 2)
+                        .addCopies(GET_TABLE, 1)
```

Comment:
❤️ 

maybe it would be wise to have an MV defined on couple source tables, so that impact is more visible in the test

(i should have thought about this in https://github.com/trinodb/trino/pull/27602/commits/37a7dca16fabe8e26092310e41d35f5a524536c9 ...)